### ISSUE TITLE
Macro "release_key" defined in "MMKV.h" pullute the global scope.

### ISSUE THREAD

**Issue by anthann** [Tencent/MMKV#1215]
### The language of MMKV
C++

### The version of MMKV
V1.3.2

### The platform of MMKV
POSIX

### The installation of MMKV
Git clone

### What's the issue?
The macro "#define release_key" in "MMKV.h" pollute the global scope and make my protobuffer generated files failed to be compile.

clang output:
pkg_manager.pb.h:4318:16: error: expected member name or ';' after declaration specifiers
  std::string* release_key();
  ~~~~~~~~~~~  ^
../../third_party/mmkv/Core/MMKV.h:111:30: note: expanded from macro 'release_key'
#  define release_key(key) ((void)0)

### What's the log of MMKV when that happened?
No log as this is a comile issue.

**Comment by lingol** [comment]
Noted.

**Comment by lingol** [comment]
Released with [v1.3.3](https://github.com/Tencent/MMKV/releases/tag/v1.3.3).
